go/bridge/bridge.go

```diff
@@ -141,6 +143,8 @@ func handler(event events.APIGatewayProxyRequest) (res Response, err error) {
 func ParseJsonIntoRequest(body string) (Request, error) {
 	var req Request
 	err := json.Unmarshal([]byte(body), &req)
+	req.RequestURI = req.Path
```

Comment:
Is this the best location and value to use?

Reply:
This place seems to make sense to me. Go seems too strict to abuse the JSON field names, so it is what it is.

Reply:
Yeah. I did try mapping `path` to `RequestURI`, but Go silently populates neither `RequestURI` or `Path` in that case, which was quite confusing.